wix/windows-connector.wxs

```diff
@@ -4,7 +4,7 @@
       Id="*"
       Name="Google Cloud Print Windows Connector"
       Language="1033"
-      Version="1.0.0.0"
+      Version="1.0.0"
```

Comment:
Since we have an unmet milestone for 1.0, and we aren't ready to release binaries yet, I think this number should be <1.0. Maybe 0.9 because it's getting close.